<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205, 205, 205);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ISBN / Judul Buku</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSearch">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid black;
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid black;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(205, 205, 205);
}

QPushButton:pressed{
padding-left: 5px;
padding-top: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../ICON/search.png</normaloff>../ICON/search.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblBuku">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>711</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Judul</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Genre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Penulis</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Penerbit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tgl Terbit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah Stok</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
